uefi-firmware.nix

```diff
@@ -59,7 +59,13 @@ let
       rev = "2c81e0fc74f703012dd3b2f18da5be256e142fe3"; # Latest on r35.3.1-updates as of 2023-05-17
       sha256 = "sha256-Qh1g+8a7ZcFG4VmwH+xDix6dpZ881HaNRE/FJoaRljw=";
     };
-    patches = edk2NvidiaPatches ++ [ ./capsule-authentication.patch ];
+    patches = edk2NvidiaPatches ++ [
+      (fetchpatch {
+        url = "https://patch-diff.githubusercontent.com/raw/NVIDIA/edk2-nvidia/pull/68.patch";
```

Comment:
Recommend using a url like this one instead:
````
https://github.com/NVIDIA/edk2-nvidia/commit/9604259b0d11c049f6a3eb5365a3ae10cfb9e6d9.patch
```
The path-diff contents might change if the branch changes.

Also, inside the patch, would we not want to keep the DEBUG message?

Reply:
I left out the debug message since the T234's behavior in this instance is to not print anything either.

Reply:
Sounds good to me!